lib/gitlab/cli_helpers.rb

```diff
@@ -134,6 +149,54 @@ def record_table(data, cmd, args)
       end
     end
 
+    # Renders the result of given commands and arguments into a Hash
+    #
+    # @param  [Array]  data         Resultset from the API call
+    # @param  [String] cmd          The command passed to the API
+    # @param  [Array]  args         Options passed to the API call
+    # @param  [bool]   single_value If set to true, a single result should be returned
+    # @return [Hash]   Result hash
+    def record_hash(data, cmd, args, single_value = false)
+      if data.empty?
+        result = nil
+      else
+        arr, keys = get_keys(args, data)
+        result = []
+        arr.each_with_index do |hash, index|
```

Comment:
Unless I've missed it, you don't actually use the index.  You should probably just use `each` instead.